test/functional/feature_block.py

```diff
@@ -1295,7 +1295,7 @@ def run_test(self):
 
         self.log.info("Reject a block with an invalid block header version")
         b_v1 = self.next_block('b_v1', version=1)
-        self.send_blocks([b_v1], success=False, force_send=True, reject_reason='bad-version(0x00000001)', reconnect=True)
+        self.send_blocks([b_v1], success=False, force_send=True, reject_reason='bad-version(0x00620001)', reconnect=True)
```

Comment:
There's no block version check on parsing the number, AFAIK, it's just down to whether the AuxPoW flag is set. I can (and am working on this now) do this in a neater way, but AuxPoW was triggered on height wasn't it?

Reply:
> AuxPoW was triggered on height wasn't it?

Correct. This is why it's strange - a non-auxpow client (e.g. 1.5 or 1.6, there's still people running these according to [blockchair](https://blockchair.com/dogecoin/nodes) would mine a block with `0x00000001`, not `0x00620001`. This is minor - I just found it a curious thing because it doesn't really reflect mainnet reality.

Reply:
Turns out to avoid this, we'd have to make the test block production code height aware, so that it sets the AuxPoW bit only once required. That's a big change, so going to presume not worth it, for now.

Reply:
> test block production code

On `blocktools` code, right? I'll just make a note and have a look once we are in a state where we have auxpow working. Consider this settled for the time being.